common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -663,7 +663,7 @@ def convert(content, input_format, output_format):
                     # the exception if something went wrong in parsing the transcript.
                     srt_subs = SubRipFile.from_string(
                         # Skip byte order mark(BOM) character
-                        content.decode('utf-8-sig'),
+                        content.decode('utf-8-sig') if six.PY2 else content,
```

Comment:
Why wouldn't we want to skip the BOM in py3? does something else automatically take care of it?

Reply:
in python 3 the `content` is of type `string` and decoding that string caused the failure, so I made decoding conditional

Reply:
I see but is the BOM still there if we do this in python 3?  Perhaps we have to do `content.encode('utf-8').decode('utf-8-sig')` in python 3 to get the signature removed here?

Reply:
I tested this after rebasing and `content` doesn't have BOM and works fine without decoding it